doc/website/src/pages/index.tsx

```diff
@@ -12,24 +12,13 @@ function HomepageHeader() {
   return (
     <header className={clsx('hero hero--primary', styles.heroBanner)}>
       <div className="container">
-        <Heading as="h1" className="hero__title">
-          {siteConfig.title}
-        </Heading>
         <div className="diagramImg">
           <img src="./img/holos-diagram-color-transparent.svg" alt="Holos Diagram" />
         </div>
+        <Heading as="h1" className="hero__title">
```

Comment:
I moved the diagram up so it was more in alignment (visually) with the title/tagline. It also looks good on mobile.